registration/views/dealers.py

```diff
@@ -311,7 +311,7 @@ def add_assistants_checkout(request):
 
     if status:
         # Payment succeeded - Mark assistants as paid
-        for assistant in dealer.dealerasst_set.all():
+        for for assistant in dealer.dealerasst_set.all().filter(paid=False):
```

Comment:
Small bug here, but otherwise looks great!  Thanks for your contribution.

Reply:
Fixed... can't do anything w/o at least one stupid error 🤣